src/main/java/frc/robot/Constants.java

```diff
@@ -222,5 +222,13 @@ public static class Vision {
     public static final Matrix<N3, N1> kLeftMultiTagStdDevs = VecBuilder.fill(0.5, 0.5, 1);
 
     public static final Pose2d target = new Pose2d(1, 1, new Rotation2d(Units.degreesToRadians(0)));
+  }
+  public static class trapConstants {
+    public static final int TRAP_MOTOR_ID = 0;
+    public static final int M_CLIMBINGRIGHT_MOTOR_ID = 1;
+    public static final int M_CLIMBINGLEFT_MOTOR_ID = 2;
```

Comment:
climbing constants should be in a different constants class

Reply:
fixed